src/zulqer/groupPeople/__test__/index.test.ts

```diff
@@ -0,0 +1,24 @@
+import { getPeopleByCity, getPeopleByGender } from '../index';
```

Comment:
Lets have folder name `__tests__` instead of `__test__`
![image](https://user-images.githubusercontent.com/71023607/93057108-069c1200-f68b-11ea-8cb3-14af6ba2bdc7.png)

Reply:
No. Let's keep unit test cases in the same directory as code.